run-tests-pw.sh

```diff
@@ -1,20 +1,23 @@
-# This file is part of DataCite.
+#!/usr/bin/env bash
+# -*- coding: utf-8 -*-
 #
-# Copyright (C) 2015, 2016 CERN.
+# This file is part of DataCite.
+# Copyright (C) 2015-2020 CERN.
 #
-# DataCite is free software; you can redistribute it and/or modify it
-# under the terms of the Revised BSD License; see LICENSE file for
-# more details.
+# Datacite is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
 
+# Run tests with real HTTP calls to Datacite REST APIs to their test endpoint.
 
 # Quit on errors
 set -o errexit
 
 # Quit on unbound symbols
 set -o nounset
 
-python -m check_manifest --ignore ".*-requirements.txt"
-python -m sphinx.cmd.build -qnNW docs docs/_build/html
-python -m pytest --runpw
-python -m sphinx.cmd.build -qnNW -b doctest docs docs/_build/doctest
+if [ ! -z DATACITE_USER ] || [ ! -z DATACITE_PW ] || [ ! -z DATACITE_PREFIX ]; then
```

Comment:
This didn't work for me. This version does:
```if [ -z "{$DATACITE_USER}" ] || [ -z "{$DATACITE_PW}" ] || [ -z "${DATACITE_PREFIX}" ]; then```

Reply:
You are totally right... I thought I tested well, but I changed multiple times I kept the wrong version. Thanks a lot!